domain-proxy/pkg/client/client.go

```diff
@@ -0,0 +1,94 @@
+package client
+
+import (
+	"fmt"
+	. "github.com/redhat-appstudio/jvm-build-service/domain-proxy/pkg/common"
```

Comment:
Using this import structure doesn't work for me in intellij hence my other restructuring thoughts :-)

Reply:
Interesting. I could simply change the module name to fix this for you.